docs/js/extra.js

```diff
@@ -51,3 +51,42 @@ function setCopyText() {
 document$.subscribe(function () {
   setCopyText();
 });
+
+// Use client-side redirects for anchors that have moved.
+// Other redirects should use `redirect_maps` in the `mkdocs.yml` file instead.
+(function () {
+  let redirect_maps = {
+    "concepts/projects/#managing-dependencies":
+      "concepts/projects/dependencies/",
+  };
+
+  // The prefix for the site, see `site_dir` in `mkdocs.yml`
+  let site_dir = "uv";
+
+  function get_path() {
```

Comment:
Is this required? Could we just do `window.location.pathname + window.location.hash` ?

Reply:
Is the pathname guaranteed to end in a `/`?

Reply:
I'm not sure, I can't find any documentation about whether this is guaranteed or not but from my experience modern browsers do tend to maintain this consistency even if you might not include a trailing `/` the browser will include it. I think it's fine to keep this in just to be on the safer side.